BitwardenShared/Core/Platform/Services/StateService.swift

```diff
@@ -1922,6 +1959,12 @@ actor DefaultStateService: StateService { // swiftlint:disable:this type_body_le
         .eraseToAnyPublisher()
     }
 
+    func trackUserAction(_ action: UserAction) async {
+        if appSettingsStore.reviewPromptData?.reviewPromptShownForVersion != appVersion {
+            appSettingsStore.addUserAction(action)
+        }
```

Comment:
🤔 I think we should consider adding this to `ReviewPromptService` and having `StateService` just expose functions to get and set `ReviewPromptData`.